src/common/utils/templateFilters.js

```diff
@@ -7,4 +7,12 @@ const uncamelCase = (value) => {
   return uncamelStr;
 };
 
+const containsError = (array, value) => {
```

Comment:
This is the logic for the aria-describedby for errors.

Goes through the error array, usually elements are an identifier/message pair, so check the identifier for a match, and return true if so. This will then add to the attribute back in the original template.